compiler/optimizer/OSRDefAnalysis.cpp

```diff
@@ -596,6 +596,16 @@ int32_t TR_OSRDefAnalysis::perform()
          }
       return 0;
       }
+   else if (comp()->getOSRMode() == TR::voluntaryOSR)
```

Comment:
This doesn't account for the liveness analysis being disabled or anything like that...